commands/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Sample Commands Extension",
+  "description": "Press Ctrl+Shift+Y to send an event (Command+Shift+Y on a Mac).",
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/commands",
+  "manifest_version": 2,
+  "version": "1.0",
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "applications": {
+    "gecko": {
+      "id": "commands@mozilla.org"
+    }
+  },
+  "commands": {
+    "toggle-feature": {
+      "suggested_key": { "default": "Ctrl+Shift+Y" },
```

Comment:
Thanks, that worked.   The shortcut "Ctrl+Shift+Y" does work for me.  I just needed to uninstall and reinstall the extension.